corehq/apps/hqwebapp/migrations/0009_truncate_authtoken_table.py

```diff
@@ -0,0 +1,19 @@
+from django.db import migrations
+
+
+def _truncate_authtoken_table(apps, schema_editor):
+    # NOTE: truncating to preserve migration records
+    from django.db import connection
+    cursor = connection.cursor()
+    cursor.execute("TRUNCATE TABLE authtoken_token")
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hqwebapp', '0008_hqoauthapplication'),
+    ]
+
+    operations = [
+        migrations.RunPython(_truncate_authtoken_table, reverse_code=migrations.RunPython.noop)
```

Comment:
This can be simplified as  `migrations.RunSQL('TRUNCATE TABLE authtoken_token')`. Similarly the reverse can be changed to `migrations.RunSQL.noop`. There are a bunch of examples of this elsewhere, if you want to double check the syntax